### PR TITLE
feat: Improving security and limits sections

### PR DESCRIPTION
**Pull request by mtrunkat**
Just trying out to edit the first two pages. After this, I jump on major refactorings :)

I changed the display of limits a bit to support the new pricing plan structure where limits will be connected with the plan:

obrazek

obrazek

### CODE REVIEW

File: sources/platform/actors/limits.md [hunk 1]
```diff
@@ -1,50 +1,134 @@
...
-| Maximum number of concurrent actor runs per user for paid accounts | 250<!-- DEFAULT_PLATFORM_LIMITS.PAID_ACCOUNT_MAX_CONCURRENT_ACTOR_RUNS_PER_USER --> |
-| Maximum number of actors per schedule                              | 10<!-- DEFAULT_PLATFORM_LIMITS.MAX_ACTORS_PER_SCHEDULER -->                         |
-| Maximum number of tasks per schedule                               | 10<!-- DEFAULT_PLATFORM_LIMITS.MAX_TASKS_PER_SCHEDULER -->                          |
+The tables below demonstrate the Apify Platform's default resource limits. For API limits such as rate limits and max payload size, see the [API documentation](https://docs.apify.com/api/v2#/introduction/rate-limiting).
+
+> If needed, the limits shown below can be increased on paid accounts. For details, contact us on **hello@apify.com** or using the chat icon in the bottom-right corner.
+
+## Actor runtime limits {#actor-limits}
+
+<table>
```
Comment (B4nan) [inline sources/platform/actors/limits.md:20]: do we really need html table?

**Comment by mnmkng** [comment]
CleanShot 2023-02-28 at 14 55 21@2x
What's a task input schema?

File: sources/platform/actors/limits.md [hunk 2]
```diff
@@ -1,50 +1,134 @@
...
-| Maximum number of webhooks per user                                | 100<!-- DEFAULT_PLATFORM_LIMITS.MAX_TASKS_PER_USER -->                              |
-| Maximum number of concurrent actor runs per user for free accounts | 25<!-- DEFAULT_PLATFORM_LIMITS.FREE_ACCOUNT_MAX_CONCURRENT_ACTOR_RUNS_PER_USER -->  |
-| Maximum number of concurrent actor runs per user for paid accounts | 250<!-- DEFAULT_PLATFORM_LIMITS.PAID_ACCOUNT_MAX_CONCURRENT_ACTOR_RUNS_PER_USER --> |
-| Maximum number of actors per schedule                              | 10<!-- DEFAULT_PLATFORM_LIMITS.MAX_ACTORS_PER_SCHEDULER -->                         |
-| Maximum number of tasks per schedule                               | 10<!-- DEFAULT_PLATFORM_LIMITS.MAX_TASKS_PER_SCHEDULER -->                          |
+The tables below demonstrate the Apify Platform's default resource limits. For API limits such as rate limits and max payload size, see the [API documentation](https://docs.apify.com/api/v2#/introduction/rate-limiting).
+
+> If needed, the limits shown below can be increased on paid accounts. For details, contact us on **hello@apify.com** or using the chat icon in the bottom-right corner.
```
Comment (mnmkng) [inline sources/platform/actors/limits.md:16]: I don't think there's a chat icon in the bottom right corner anymore. Or does it automatically appear on the docs when you're a paying user?

File: sources/platform/actors/limits.md (hunk 1 above)
Comment (mtrunkat) [reply sources/platform/actors/limits.md:20]: without that you can't use colspan and rowspan which is quite needed now with the new billing limits

File: sources/platform/actors/limits.md [hunk 3]
```diff
@@ -1,50 +1,134 @@
 ---
 title: Limits
-description: Learn the Apify platform's resource capability and limitations such as max memory, disk size and number of actors/tasks per user or schedule.
+description: Learn the Apify platform's resource capability and limitations such as max memory, disk size and number of actors and tasks per user.
```
Comment (TheoVasilis) [inline sources/platform/actors/limits.md:3]: Learn the Apify platform's resource capability and limitations, such as max memory, disk size, and the number of actors and tasks per user.

File: sources/platform/actors/limits.md [hunk 4]
```diff
@@ -1,50 +1,134 @@
...
 sidebar_position: 7.8
 slug: /actors/limits
 ---
 
 # Limits {#limits}
 
-**Learn the Apify platform's resource capability and limitations such as max memory, disk size and number of actors/tasks per user or schedule.**
+**Learn the Apify platform's resource capability and limitations such as max memory, disk size and number of actors and tasks per user.**
```
Comment (TheoVasilis) [inline sources/platform/actors/limits.md:10]: Learn the Apify platform's resource capability and limitations, such as max memory, disk size, and the number of actors and tasks per user.

File: sources/platform/actors/security.md [hunk 5]
```diff
@@ -11,6 +11,24 @@ slug: /actors/security
...
+
+<a href="https://apify.com/security-whitepaper.pdf" target="_blank" title="Apify Security Whitepaper">
+    <img src={require("./images/security/whitepaper-cover.png").default} width="50%" title="Apify Security Whitepaper" />
+</a>
+
+## Reporting a vulnerability
+
+If you notice or suspect a potential security breach, please report this immediately to our security team at [security@apify.com](mailto:security@apify.com), including all the technical details.
+
+The total compensation for your report will be highly dependent on the severity, complexity, and exploitability of the vulnerability. Your reports are always verified by our internal security team,
```
Comment (TheoVasilis) [inline sources/platform/actors/security.md:27]: The total compensation for your report will be highly dependent on the vulnerability's severity, complexity, and exploitability. Our internal security team always verifies your reports.

File: sources/platform/actors/security.md [hunk 6]
```diff
@@ -11,6 +11,24 @@ slug: /actors/security
...
+which also weighs in on various risks associated with that vulnerability and how major of an impact it would have in case we were attacked.
+
+We strive to reply to all reports within 5 working days. However, depending on the complexity of the issue or our current workload, our response might take longer.
+
+## Securing your data
 
-Read the [Apify Security Whitepaper](https://apify.com/security-whitepaper.pdf) for a full description of Apify's measures and commitments to security.
+Apify Platform provides you with multiple ways to secure your data. This includes [encrypted environment variables](./development/environment-variables) to store your configuration secrets and [encrypted input](./development/secret-input) to secure the input parameters of your actors.
```
Comment (TheoVasilis) [inline sources/platform/actors/security.md:34]: The Apify platform provides multiple ways to secure your data. This includes [encrypted environment variables](./development/environment-variables) to store your configuration secrets and [encrypted input](./development/secret-input) to secure the input parameters of your actors.

**Review by TheoVasilis — CHANGES_REQUESTED** [review]

**Pull request merged**